plugins/woocommerce/src/Internal/Orders/SourceAttributionController.php

```diff
@@ -0,0 +1,473 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Internal\Orders;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController;
+use Automattic\WooCommerce\Internal\Features\FeaturesController;
+use Automattic\WooCommerce\Internal\RegisterHooksInterface;
+use Automattic\WooCommerce\Internal\Traits\ScriptDebug;
+use Automattic\WooCommerce\Internal\Traits\SourceAttributionMeta;
+use Automattic\WooCommerce\Proxies\LegacyProxy;
+use Exception;
+use WC_Customer;
+use WC_Log_Levels;
+use WC_Logger_Interface;
+use WC_Order;
+use WC_Tracks;
+use WP_User;
+
+/**
+ * Class SourceAttributionController
+ *
+ * @since x.x.x
+ *
+ * phpcs:disable WordPress.PHP.DisallowShortTernary
```

Comment:
Similar to an earlier question, why not follow the rule (it doesn't seem necessary to make an exception)?

Reply:
Same answer as here https://github.com/woocommerce/woocommerce/pull/39701#discussion_r1388333589

Reply:
Removed (https://github.com/woocommerce/woocommerce/pull/39701/commits/60955d44b02ae3a339a2e83d9e20b33fca6b93fb)